docs/testing/testing_in_nx/frontend_testing_in_nx.md

```diff
@@ -57,9 +56,7 @@ If you are considering adding visual regression tests to your application, then
     - A powerful visual regression testing tool that detects visual changes, updates of browsers or OS, and integrates with popular testing frameworks and CI/CD pipelines, making it easy to automate visual regression testing and improve the quality and user-friendliness of digital products. Our community of experts recommend this software as the best approach, to decide if it is right for you, please visit [Applitools Eyes!](https://applitools.com/platform/eyes/)
 - Native
     - A free alternative to cloud-based providers which enables developers to perform automated visual regression testing using the latest web platform features in their chosen testing framework.
-
-:::note
-
-We do not currently support visual testing with Ensono Stacks using our `@ensono-stacks/cypress` plugin
-
-:::
+<br></br>

```

Comment:
Are these required? Can they be deleted?

Reply:
Not really - i just put in to give a bit more space between the footer and the main page to make it look nice, but that is subjective.  I have removed the line breaks.
With
![image](https://github.com/user-attachments/assets/53a8dd8e-832d-4748-bfb2-419700706c85)
Without
Without
![image](https://github.com/user-attachments/assets/20b5ae46-9566-4f9a-9fd0-4c61fd006086)